.teamcity/src/subprojects/release/publishing/ProjectPublishing.kt

```diff
@@ -0,0 +1,28 @@
+package subprojects.release.publishing
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import subprojects.*
+import subprojects.build.core.*
+
+data class Build(val name: String, val build: BuildType?)
+
+object ProjectPublishing : Project({
+    id("ProjectPublishing")
+    name = "Publishing"
+    description = "Publish artifacts to repositories"
+
+    val publishingTargets = listOf(
```

Comment:
This and the following line could be merge in one cause you don't use `publishingTargets` anywhere else